.github/workflows/tests.yaml

```diff
@@ -0,0 +1,45 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-versions: ['7.3', '7.4']
```

Comment:
1. Could you please explain why we now have PHP 7.3, 7.4 builds both on Travis and GH Actions?
2. Did you introduce GH Actions to be able to test PHP 8.0? Or is it the start of transitioning from Travis in this PR?

Reply:
1. I'm adding 7.3 here to make sure I don't accidentally break something on 7.3, as being able to test all versions on GH actions is valuable due to their swiftness. 
2. Exactly, to test on PHP 8.0. Travis CI's builds depend on Box, but it isn't coming yet. We have a very good and solid Travis CI pipeline, I don't think it is time to throw it away.